magefile.go

```diff
@@ -99,6 +99,10 @@ func Update() error {
 	return sh.Run("make", "update")
 }
 
+func Fields() error {
+	return mage.GenerateFieldsYAML(".")
```

Comment:
I think this should be `model` rather than `.`. 

(I think the docs that I wrote in `GenerateFieldsYAML` are not 100% accurate w.r.t. how it searches for fields.yml files because I didn't fully understand the subtleties of how global_fields/main.go searches.)

Reply:
that won't skip `_meta/fields.common.yml`?

Reply:
`_meta/fields.common.yml` will still be included. The args to `GenerateFieldsYAML` are extra module directories to collect fields.yml.

Reply:
We will do a follow up fix then.